assets/evm/v1/assets_dev.json

```diff
@@ -11,5 +11,445 @@
+            }
+        ]
+    },
+    {
+        "symbol": "BUSD",
+        "precision": 18,
+        "name": "Binance-Peg BUSD Token",
+        "priceId": "binance-usd",
+        "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/tokens/white/BUSD.svg",
+        "instances": [
+            {
+                "chainId": "fe58ea77779b7abda7da4ec526d14db9b1e9cd40a217c34892af80a9b332b76d",
+                "contractAddress": "0xA649325Aa7C5093d12D6F98EB4378deAe68CE23F"
+            }
+        ]
+    },
+    {
+        "symbol": "MATIC",
+        "precision": 18,
+        "name": "Matic",
+        "priceId": "matic-network",
+        "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/tokens/white/MATIC.svg",
+        "instances": [
+            {
+                "chainId": "fe58ea77779b7abda7da4ec526d14db9b1e9cd40a217c34892af80a9b332b76d",
+                "contractAddress": "0x3405A1bd46B85c5C029483FbECf2F3E611026e45"
+            }
+        ]
+    },
+    {
+        "symbol": "DAI",
+        "precision": 18,
+        "name": "Dai Stablecoin",
+        "priceId": "dai",
+        "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/tokens/white/DAI.svg",
+        "instances": [
+            {
+                "chainId": "fe58ea77779b7abda7da4ec526d14db9b1e9cd40a217c34892af80a9b332b76d",
+                "contractAddress": "0x765277EebeCA2e31912C9946eAe1021199B39C61"
+            }
+        ]
+    },
+    {
+        "symbol": "AVAX",
+        "precision": 18,
+        "name": "Avalanche",
+        "priceId": "avalanche-2",
+        "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/tokens/white/AVAX.svg",
+        "instances": [
+            {
+                "chainId": "fe58ea77779b7abda7da4ec526d14db9b1e9cd40a217c34892af80a9b332b76d",
+                "contractAddress": "0x4792C1EcB969B036eb51330c63bD27899A13D84e"
+            }
+        ]
+    },
+    {
+        "symbol": "WBTC",
+        "precision": 8,
+        "name": "Wrapped BTC",
+        "priceId": "wrapped-bitcoin",
+        "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/tokens/white/WBTC.svg",
+        "instances": [
+            {
+                "chainId": "fe58ea77779b7abda7da4ec526d14db9b1e9cd40a217c34892af80a9b332b76d",
+                "contractAddress": "0x922D641a426DcFFaeF11680e5358F34d97d112E1"
+            }
+        ]
+    },
+    {
+        "symbol": "WBTC",
+        "precision": 8,
+        "name": "Wrapped BTC",
+        "priceId": "wrapped-bitcoin",
+        "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/tokens/white/WBTC.svg",
+        "instances": [
+            {
+                "chainId": "fe58ea77779b7abda7da4ec526d14db9b1e9cd40a217c34892af80a9b332b76d",
+                "contractAddress": "0xE57eBd2d67B462E9926e04a8e33f01cD0D64346D"
+            }
+        ]
```

Comment:
Same - lets leave a single WBTC for one chain

Reply:
done